lib/rules/at-rule-no-deprecated/__tests__/index.mjs

```diff
@@ -67,15 +67,6 @@ testRule({
 				text: '',
 			},
 		},
-		{
-			code: 'a { @apply foo; }',
```

Comment:
[suggestion] Can you add an `accept` case for `@apply`, in addition to this removal?